.gitignore

```diff
@@ -145,18 +154,26 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+# PyCharm
+#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+.idea/
```

Comment:
I have a question, in the commend part added it says that is not recommended to uncomment ".idea/", are we looking for a more nuclear option in this case?

Reply:
The `.idea/` directory already existed in our `.gitignore`, but since it was introduced in the official python `.gitignore` file, we just moved it to this section.